extensions/vscode/src/types/shared.ts

```diff
@@ -8,9 +8,7 @@ import {
 } from "../api";
 
 export type DeploymentSelector = {
-  deploymentName: string;
-  projectDir: string;
-  configurationName?: string;
+  deploymentPath: string;
```

Comment:
The deploymentPath value is guaranteed to be unique across the user's installation. This is set by the agent while reading the file system. Once we match up to the applicable contentRecord, we then use information within it to find the configuration and projectDir, thus switching back to a relative access which works for all users, even if sharing files.